.github/actions/multi-approvers/__tests__/main.test.js

```diff
@@ -0,0 +1,202 @@
+/**
+ * Copyright 2024 The Authors (see AUTHORS file)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const core = require('@actions/core');
+const github = require('@actions/github');
+
+class FakeOctokit {
+  static EMPTY_MODEL = {
+    pulls: [],
+    teams: [],
+  };
+
+  model;
+
+  constructor(model) {
+    // The logic in this class depends on at least the EMPTY_MODEL being
+    // defined.
+    //
+    // Note that the splat operator is not recursive, if more than
+    // top-level members are required, a recursive application of the splat
+    // operator will have to be used.
+    this.model = {...FakeOctokit.EMPTY_MODEL, ...model};
+  }
+
+  paginate(fn, ...args) {
+    return fn(...args);
+  }
+
+  rest = {
+    pulls: {
+      get: ({owner, repo, pull_number}) => {
+        const data = this.model.pulls.find(
+            (v) => v.owner === owner && v.repo === repo &&
+                v.pull_number === pull_number);
+        if (data) {
+          return {
+            data,
+          };
+        }
+        throw {status: 404};
+      },
+      listReviews:
+          ({owner, repo, pull_number}) => {
+            const {data} = this.rest.pulls.get({owner, repo, pull_number});
+            return data.reviews || [];
+          },
+    },
+    teams: {
+      getMembershipForUserInOrg:
+          ({org, team_slug, username}) => {
+            const data =
+                (this.model.teams || [])
+                    .find(
+                        (v) => v.org === org && v.team_slug === team_slug &&
+                            v.username === username);
+            if (data) {
+              return {
+                data,
+              };
+            }
+            throw {status: 404};
+          },
+    },
+  };
+}
+
+jest.mock('@actions/core');
```

Comment:
There's a built-in mocking and testing package in the stdlib. I don't think we need/want jest or any other testing framework like sinon.

Reply:
I'm happy to use whatever library you think is best (I chose jest because it seemed to be pretty standard in the JS community). Can you point me to the package that you're referring to? I couldn't find it on a quick Google search.

Reply:
You can find examples in google-github-actions

Reply:
Done.